src/plugins/github/fetchGithubRepo.js

```diff
@@ -6,7 +6,6 @@
 
 import Database from "better-sqlite3";
 import fetch from "isomorphic-fetch";
-import retry from "retry";
```

Comment:
This is the only usage of "retry" -- remove the dependency?

Reply:
Huh, so it is. I just assumed that Discourse and Discord would also be
using it. Removed; thanks.